runtime/vm/ClassInitialization.cpp

```diff
@@ -271,7 +271,10 @@ performVerification(J9VMThread *currentThread, J9Class *clazz)
 		}
 	}
 done:
+#if defined(J9VM_OPT_INCLUDE_CLASSVERIFY_EXIT_TRACEPOINT)
+	/* This tracepoint can introduce startup regressions even when disabled, so don't build into the VM by default */
```

Comment:
Please fold this line and finish the sentence with a period.

Reply:
Maybe "split" would have been a better word than "fold" (it's too long if you can't see all of it without scrolling horizontally here).